app/webpacker/controllers/flatpickr_controller.js

```diff
@@ -60,6 +63,10 @@ export default class extends Flatpickr {
 
   open() {
     this.fp.element.dispatchEvent(new Event("focus"));
+    if (!this.defaultDateValue) return;
```

Comment:
I'd delete this line to which seems to me useless ; or maybe as the first line in `setDefaultDateValue()` method?